app/src/main/java/com/gdsctsec/smartt/ui/main/adapter/SubjectsAdapter.kt

```diff
@@ -52,61 +61,141 @@ class SubjectsAdapter(val subjectList: MutableList<String>, val timeList: Mutabl
         holder.subjectTextView.text = subjectList.get(position)
         holder.timeTextView.text = timeList.get(position)
 
-
-        when (position % 6) {
-            0 -> {
-                holder.itemView.background.setTint(
-                    ContextCompat.getColor(
-                        holder.itemView.context,
-                        R.color.faint_pink
+        if (isOverList.get(position) == false) {
+            when (position % 6) {
+                0 -> {
+                    holder.itemView.background.setTint(
+                        ContextCompat.getColor(
+                            holder.itemView.context,
+                            R.color.faint_pink
+                        )
                     )
-                )
-            }
-
-            1 -> {
-                holder.itemView.background.setTint(
-                    ContextCompat.getColor(
-                        holder.itemView.context,
-                        R.color.faint_yellow
+                }
+
+                1 -> {
+                    holder.itemView.background.setTint(
+                        ContextCompat.getColor(
+                            holder.itemView.context,
+                            R.color.faint_yellow
+                        )
                     )
-                )
-            }
-
-            2 -> {
-                holder.itemView.background.setTint(
-                    ContextCompat.getColor(
-                        holder.itemView.context,
-                        R.color.faint_neon_blue
+                }
+
+                2 -> {
+                    holder.itemView.background.setTint(
+                        ContextCompat.getColor(
+                            holder.itemView.context,
+                            R.color.faint_neon_blue
+                        )
                     )
-                )
+                }
+
+                3 -> {
+                    holder.itemView.background.setTint(
+                        ContextCompat.getColor(
+                            holder.itemView.context,
+                            R.color.faint_dark_blue
+                        )
+                    )
```

Comment:
Create a List of colors.
Then pass the color to it as

```kotlin
holder.itemView.background.setTint(
                        ContextCompat.getColor(
                            holder.itemView.context,
                            colors[position%6]
                        )
                    )
```

instead of this big when statement